packages/api/src/external/aws/app-config.ts

```diff
@@ -107,6 +108,20 @@ export async function getOidsWithIHEGatewayV2Enabled(): Promise<string[]> {
   return getCxsWithFeatureFlagEnabled("oidsWithIHEGatewayV2Enabled");
 }
 
+export async function getE2eCxIds(): Promise<string | undefined> {
+  if (Config.isDev()) {
+    const apiKey = getEnvVar("TEST_API_KEY");
+    return apiKey ? getCxIdFromApiKey(apiKey) : undefined;
+  }
+  const cxIds = await getCxsWithFeatureFlagEnabled("e2eCxIds");
+  if (cxIds.length > 0) {
+    const msg = `FF e2eCxIds should only have 1 cxId`;
+    log(`${msg} but it has ${cxIds.length}, using the first one.`);
+    capture.message(msg, { extra: { cxIds }, level: "warning" });
+  }
+  return cxIds[0];
+}
```

Comment:
I recommend we consolidate FF code on these functions, avoiding `isDev()` to use an env var outside of the respective function.